### PR TITLE
Remove .characters because of deprecation

### PR DESCRIPTION
**Pull request by alirp88**

### CODE REVIEW

**Comment by 0xTim** [comment]
@alirp88 Vapor 2 still needs to work with Swift 3 so we can't break the codebase by removing the `.characters` property. The only way to do it would be to add both sets of code wrapped in `#if swift(>=4)` macros

File: Sources/Debugging/Debuggable.swift [hunk 1]
```diff
@@ -105,11 +105,17 @@ extension Debuggable {
 
 extension String {
     func readableTypeName() -> String {
+    #if swift(>=4.0)
+        let characterSequence = self
+            .split(separator: ".")
```
Comment (0xTim) [inline Sources/Debugging/Debuggable.swift]: I would end the `#else` after the first line - no point duplicating more code than we have to

**Comment by 0xTim** [comment]
@tanner0101 We happy with this fix for removing the deprecation warnings?

**Comment by alirp88** [comment]
@0xTim I will fix other PRs too if this will be ok for you guys

**Comment by alirp88** [comment]
let me fix this, it has error!

**Comment by 0xTim** [comment]
Cool if you can get the build to go through that would be awesome. I'd hold off on the other PRs until @tanner0101 comes online just to make sure he is happy with the way of doing it, just in case there is a reason that it hasn't been done yet. Though I suspect this is because the focus has been on Vapor 3

**Comment by alirp88** [comment]
Fixed now,

**Comment by tanner0101** [comment]
we'll fix ci when we break out the vapor 3 packages. merge and testing locally for now. thanks guys!

**Pull request merged**